cms/templates/program_authoring.html

```diff
@@ -1,15 +1,16 @@
+<%page expression_filter="h"/>
 <%namespace name='static' file='static_content.html'/>
 <%! from django.utils.translation import ugettext as _ %>
 
 <%inherit file="base.html" />
 <%block name="title">${_("Program Administration")}</%block>
 
 <%block name="header_extras">
-<%static:css group='style-edx-icons'/>
 <link rel="stylesheet" href=${authoring_app_config.css_url}>
 </%block>
 
 <%block name="requirejs">
+  ## safe-lint: disable=mako-invalid-js-filter
```

Comment:
@cahrens @robrap because I tried to fix as Robert guided, but it threw this error
https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/21975/testReport/cms.djangoapps.contentstore.views.tests.test_programs/TestProgramAuthoringView/test_authoring_header/

search for js_escaped_string(authoring_app_config.js_url